plugins/org.eclipse.elk.alg.packing.rectangles/src/org/eclipse/elk/alg/packing/rectangles/rectPacking.melk

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Kiel University and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Kiel University - initial API and implementation
+ *******************************************************************************/ 
+package org.eclipse.elk.alg.packing.rectangles
+ 
+/* Metadata file for rectangle packing algorithms. Enables support for options and specifies auto-generated files.
+ * 
+ * @author dalu
+ */
+
+import org.eclipse.elk.alg.packing.rectangles.RectPackingLayoutProvider
+import org.eclipse.elk.alg.packing.rectangles.util.PackingStrategy
+import org.eclipse.elk.core.math.ElkPadding
+
+//BUNDLE
+bundle {
+    metadataClass options.RectPackingMetaDataProvider
+}
```

Comment:
Add `idPrefix org.eclipse.elk.rectpacking` (or similar) to avoid `alg` within the option ids.